meta/documents/user_guide/en/plugin_installation.md

```diff
@@ -0,0 +1,45 @@
+# Wayfair plugin: Installation instructions
+
+1. Login to the Plentymarkets system
+
+2. Go to **plugins >> plentyMarketplace**, which will use the system owner’s credentials to log you in on the Marketplace
+
+3. Click the search button  in the top-right corner of the page
+
+4. Enter “Wayfair” into the box and press enter

```

Comment:
I'm leaning towards keeping these two steps as is regardless but could we tell users to just go to this link or is this something likely to change with version or some other reason: https://marketplace.plentymarkets.com/en/plugins/integration/wayfair_6273

Reply:
I thought about this as well. The problem with the link is that we're not in control of it. Plenty could change that link at any point without telling us.
Also, the user may click the link **before using the plentyMarketplace link in Plentymarkets** and therefore be unauthorized, or **they even could be authorized for the wrong system**.